storagenode/console/consoleserver/server.go

```diff
@@ -127,163 +105,106 @@ func (server *Server) Close() error {
-	// if satelliteID != nil {
-	// 	satelliteStats, err := server.service.GetStatsFromSatellite(ctx, *satelliteID)
-	// 	if err != nil {
-	// 		return response, err
-	// 	}
-	//
-	// 	response.UptimeCheck = satelliteStats.UptimeCheck
-	// 	response.AuditCheck = satelliteStats.AuditCheck
-	// }
-
-	response.DiskSpace = *space
-	response.Bandwidth = *usage
-	response.WalletAddress = walletAddress
-	response.VersionInfo = versionInfo
-	response.IsLastVersion = true
-	response.Uptime = uptime
-	response.NodeID = nodeID.String()
-	response.Satellites = satellites
-	response.BandwidthChartData = bandwidthChartData
-	//response.DiskSpaceChartData = diskSpaceChartData
-
-	return response, nil
-}
+	if err = server.service.VerifySatelliteID(ctx, satelliteID); err != nil {
+		server.writeError(wr, http.StatusNotFound, Error.Wrap(err))
+		return
+	}
 
-func (server *Server) getBandwidth(ctx context.Context, satelliteID *storj.NodeID) (_ *console.BandwidthInfo, err error) {
-	if satelliteID != nil {
-		return server.service.GetBandwidthBySatellite(ctx, *satelliteID)
+	data, err := server.service.GetSatelliteData(ctx, satelliteID)
+	if err != nil {
+		server.writeError(wr, http.StatusInternalServerError, Error.Wrap(err))
+		return
 	}
 
-	return server.service.GetUsedBandwidthTotal(ctx)
+	server.writeData(wr, data)
 }
 
-func (server *Server) getBandwidthChartData(ctx context.Context, satelliteID *storj.NodeID) (_ []console.BandwidthUsed, err error) {
-	from, to := date.MonthBoundary(time.Now().UTC())
+// jsonOutput defines json structure of api response data.
+type jsonOutput struct {
+	Data  interface{} `json:"data"`
+	Error string      `json:"error"`
+}
 
-	if satelliteID != nil {
-		return server.service.GetDailyBandwidthUsed(ctx, *satelliteID, from, to)
-	}
+// writeData is helper method to write JSON to http.ResponseWriter and log encoding error.
+func (server *Server) writeData(wr http.ResponseWriter, data interface{}) {
+	wr.Header().Set(contentType, applicationJSON)
+	wr.WriteHeader(http.StatusOK)
 
-	return server.service.GetDailyTotalBandwidthUsed(ctx, from, to)
-}
+	output := jsonOutput{Data: data}
 
-func (server *Server) getStorage(ctx context.Context, satelliteID *storj.NodeID) (_ *console.DiskSpaceInfo, err error) {
-	if satelliteID != nil {
-		return server.service.GetUsedStorageBySatellite(ctx, *satelliteID)
+	if err := json.NewEncoder(wr).Encode(output); err != nil {
+		server.log.Error("json encoder error", zap.Error(err))
 	}
-
-	return server.service.GetUsedStorageTotal(ctx)
 }
 
-func (server *Server) checkSatelliteID(satelliteIDs storj.NodeIDList, satelliteID storj.NodeID) error {
-	for _, id := range satelliteIDs {
-		if satelliteID == id {
-			return nil
-		}
-	}
+// writeError writes a JSON error payload to http.ResponseWriter log encoding error.
+func (server *Server) writeError(wr http.ResponseWriter, status int, err error) {
+	server.log.Error("api handler error", zap.Int("status code", status), zap.Error(err))
```

Comment:
We don't want to log as an error every HTTP status error, for example, `421`, `401` are client error, not a server error.

Reply:
missed the second part somehow:
`Log it inside of server.writeError method doing an if status == http.StatusInternalServerError { log...... }`